src/components/events/create-event/advanced-options/AddIngredient.js

```diff
@@ -0,0 +1,175 @@
+    <div style={{ marginTop: "55px", marginBottom: "25px" }}>
+      <Typography style={{ marginBottom: "10px" }}>
+        Request for guests to bring ingredients
+      </Typography>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          border: "2px solid #f0f0f0",
+          borderRadius: "15px",
+        }}
+      >
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+            width: "100%",
+          }}
+        >
+          <div
+            style={{
+              display: "flex",
+              width: "100%",
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            <div className="createFormInputDiv" style={{ width: "30%" }}>
+              <input
+                type="text"
+                name="name"
+                value={formInput.name}
+                onChange={handleChange}
+                placeholder="Ingredient"
+              />
+            </div>
+
+            <div
+              className="createFormInputDiv"
+              style={{ width: "10%", marginLeft: "2%" }}
+            >
+              <input
+                type="numbers"
+                name="quantity"
+                value={formInput.quantity}
+                onChange={handleChange}
+                min="0"
+                step="0.01"
+                placeholder="Quantity"
+              />
+            </div>
+
+            <div className="eventTimeDiv" style={{ marginLeft: "2%" }}>
+              <label>
+                Measurement
+                <Select
+                  name="measurement"
+                  value={formInput.measurement}
+                  onChange={handleChange}
+                  id="greenSelect"
+                  disableUnderline={true}
+                >
+                  <MenuItem value=""></MenuItem>
+                  <MenuItem value={"cup"}>cup</MenuItem>
+                  <MenuItem value={"cups"}>cups</MenuItem>
+                  <MenuItem value={"gallon"}>gallon</MenuItem>
+                  <MenuItem value={"gallons"}>gallons</MenuItem>
+                  <MenuItem value={"lbs"}>lbs</MenuItem>
+                  <MenuItem value={"ounce"}>ounce</MenuItem>
+                  <MenuItem value={"ounces"}>ounces</MenuItem>
+                  <MenuItem value={"pint"}>pint</MenuItem>
+                  <MenuItem value={"pints"}>pints</MenuItem>
+                  <MenuItem value={"quart"}>quart</MenuItem>
+                  <MenuItem value={"quarts"}>quarts</MenuItem>
+                  <MenuItem value={"teaspoon"}>teaspoon</MenuItem>
+                  <MenuItem value={"teaspoons"}>teaspoons</MenuItem>
+                  <MenuItem value={"tablespoon"}>tablespoon</MenuItem>
+                  <MenuItem value={"tablespoon"}>tablespoons</MenuItem>
```

Comment:
Please make the text displayed for these as Proper Case (capitalize first letter).